admin/desk/views.py

```diff
@@ -0,0 +1,74 @@
+from django.shortcuts import render
+from django.views.generic import ListView, DetailView
+
+from website.project.model import User
+
+from admin.base.utils import OSFAdmin
+from admin.desk.utils import DeskClient, DeskError
+
+
+class DeskCaseList(OSFAdmin, ListView):
+    template_name = 'desk/cases.html'
+    ordering = 'updated_at'
+    context_object_name = 'cases'
+    paginate_by = 100
+    paginate_orphans = 5
+
+    def dispatch(self, request, *args, **kwargs):
+        try:
+            return super(DeskCaseList, self).dispatch(request, *args, **kwargs)
+        except DeskError as e:
+            return render(request, 'desk/desk_error.html',
```

Comment:
The context is the part of the render. The status in the context is passed in as `status` to the template renderer and then when the renderer sees `status` it puts that string in there. See the next following lines for the context.